app/controllers/api_controller/chargeback.rb

```diff
@@ -0,0 +1,45 @@
+class ApiController
+  module Chargeback
+    # Request
+    # {
+    #   "action":"assign",
+    #
+    #   "category": "tenant",
+    #   "c_id": 1
+    # }
+    def chargebacks_assign_resource(object, _type, _id, data)
+      klass = Object.const_get(data['category'].camelcase)
+      obj = klass.find(data['c_id'])
+
+      ChargebackRate.set_assignments object.rate_type, [{:cb_rate => object, :object => obj}]
+
+      {:tags => object.tags}
+    end
+
+    # Request
+    # {
+    #   "action":"unassign",
+    #
+    #   "category": "tenant",
+    #   "c_id": 1
+    # }
+    def chargebacks_unassign_resource(object, _type, _id, data = nil)
+      obj_name = data['category']
+      obj_id   = data['c_id']
+      tag_name = "/chargeback_rate/assigned_to/#{obj_name}/id/#{obj_id}"
+
+      object.tags = object.tags.reject { |t| t.name == tag_name }
+
+      {:tags => object.tags}
+    end
+
+    def update_chargebacks
```

Comment:
No, the problem is, ChargebackTag doesn't act like a Tag. We need to use *chargebacks_(un)assign_resource*